micropython/drivers/sensor/sht3x/sht3x.py

```diff
@@ -0,0 +1,26 @@
+# SHT3x driver for MicroPython on ESP8266/ESP32
```

Comment:
Doesn't look like this is specific to ESP as it uses the common i2c API.

Can you also add that it's a temp/humidity sensor, and made by Sensirion.